### PR TITLE
add integration tests

### PR DESCRIPTION
**Pull request by rokroskar**
start adding some integration tests

- [x] add basic login test
- [x] get tests working on travis 

---

addresses #135 https://github.com/SwissDataScienceCenter/renga/issues/146

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,8 +50,11 @@ before_install:
 install:
   - ./scripts/travis-install.sh
 
-before_script:
-  - make start
+before_script: |-
+    make start
+    until sleep 5; curl --retry-max-time 1 http://gitlab.renga.local/help;
```
[the remark this answers is not in the document]
Comment (rokroskar) [reply .travis.yml]: We should redo the wait script so it actually responds to useful heartbeats

File: .travis.yml [hunk 2]
```diff
@@ -17,20 +17,24 @@
 # limitations under the License.
 
 sudo: true
-dist: xenial
+dist: trusty
```
Comment (leafty) [inline .travis.yml]: why are we going back in time?

Comment (rokroskar) [reply .travis.yml]: because xenial is not officially supported by travis and builds were failing (I've switched it on master already)

**Comment by rokroskar** [comment]
👍 on making the `Makefile` more readable...

File: .travis.yml (hunk 2 above)
Comment (leafty) [reply .travis.yml]: :+1: , for info: https://docs.travis-ci.com/user/reference/overview/

**Pull request merged**